src/datafactory/HISTORY.rst

```diff
@@ -2,6 +2,9 @@
 
 Release History
 ===============
+1.0.3
+* Support new features in ADF
```

Comment:
Can you give more detailed history notes about what new features have been added